tests/e2e/specs/out-of-game/localization.spec.js

```diff
@@ -72,3 +72,50 @@ describe('Localization', () => {
     checkLobbyTranslation(fr);
   });
 });
+
+describe('language files', () => {
+  function extractKeys(obj, keyList = []) {
+    for (let key in obj) {
+      if (typeof obj[key] === 'object' && obj[key] !== null) {
+        extractKeys(obj[key], keyList);
+      } else {
+        keyList.push(key);
+      }
+    }
```

Comment:
I think it may be more readable and faster to use Object.keys and forEach here, but I'm not 100% sure on that

```suggestion
    Object.keys(obj).forEach(key => {
      if (typeof obj[key] === 'object' && obj[key] !== null) {
        extractKeys(obj[key], keyList);
      } else {
        keyList.push(key);
      }
    });
```